runtime/gc_vlhgc/CopyForwardScheme.cpp

```diff
@@ -1768,8 +1768,8 @@ void
 MM_CopyForwardScheme::flushCache(MM_EnvironmentVLHGC *env, MM_CopyScanCacheVLHGC *cache)
 {
 	Assert_MM_false(cache->isSplitArray());
-	if(0 == (cache->flags & J9VM_MODRON_SCAVENGER_CACHE_TYPE_COPY)) {
-		if(0 == (cache->flags & J9VM_MODRON_SCAVENGER_CACHE_TYPE_CLEARED)) {
+	if(0 == (cache->flags & OMR_SCAVENGER_CACHE_TYPE_COPY)) {
+		if(0 == (cache->flags & OMR_SCAVENGER_CACHE_TYPE_CLEARED)) {
```

Comment:
missing space after 'if'

Reply:
Also, it should use `OMR_ARE_NO_BITS_SET()`.

Reply:
> Also, it should use `OMR_ARE_NO_BITS_SET()`.

Bit functions like this are not used in GC code. I am not sure we should start to use them now

Reply:
They are even used in this file, see line 2622, for example.

Reply:
I already stated our current view on when to use these macros in a previous comment (https://github.com/eclipse-openj9/openj9/pull/16820#issuecomment-1470802055) and you said you'd respect that.

The example you refer to now fits within VM flags, while this PR deals with a GC specific flag.

Reply:
Still disappointed, but I did say that.